ucsb-cs156-f23/proj-happycows-f23-5pm-3
# Made Instructor Report Improvements

## Overview

In this PR, I formatted all dollar amounts as USD currency and right aligned them in their cells.

## Screenshot
Before:
<img width="1383" alt="Screen Shot 2023-11-24 at 7 26 00 PM" src="https://github.com/ucsb-cs156-f23/proj-happycows-f23-5pm-3/assets/46539388/491e7eed-92dc-41cf-862e-272f77f79cf2">

After
<img width="1411" alt="Screen Shot 2023-11-25 at 3 20 31 PM" src="https://github.com/ucsb-cs156-f23/proj-happycows-f23-5pm-3/assets/46539388/bbcb4acb-e620-4684-9040-afb9377c344b">

https://ucsb-cs156-f23.github.io/proj-happycows-f23-5pm-3/prs/21/storybook/?path=/story/components-reports-reportheadertable--one-report

https://ucsb-cs156-f23.github.io/proj-happycows-f23-5pm-3/prs/21/storybook/?path=/story/components-reports-reportlinetable--one-report

https://ucsb-cs156-f23.github.io/proj-happycows-f23-5pm-3/prs/21/storybook/?path=/story/components-reports-reporttable--three-reports

## Validation (Optional)

1. Download this branch.
2. Run the project.
3. Go to Manage Jobs.
4. Generate Instructor Report.
5. Go to Instructor Reports.
6. Click any of the Instruction Reports.
7. Verify improvments.

## Tests


## Linked Issues

Closes #1
